packages/algob/src/internal/deployer.ts

```diff
@@ -878,6 +883,52 @@ export class DeployerDeployMode extends DeployerBasicMode implements Deployer {
 		this.registerSSCInfo(cpKey, sscInfo);
 		return sscInfo;
 	}
+	/**
+	 * Execute single transaction or group of transactions (atomic transaction)
+	 * executes `ExecParams` or `Transaction` Object, SDK Transaction object passed to this function
+	 * will be signed and sent to network. User can use SDK functions to create transactions.
+	 * Note: If passing transaction object a signer/s must be provided.
+	 * @param deployer Deployer
+	 * @param transactionParam transaction parameters or atomic transaction parameters
+	 * https://github.com/scale-it/algo-builder/blob/docs/docs/guide/execute-transaction.md
+	 * or TransactionAndSign object(SDK transaction object and signer parameters)
+	 */
+	async executeTx(
```

Comment:
How about move `executeTx` method to `DeployerBasicMode`? Because `DeployerRunMode` and `DeployerDeployerMode` both use same `executeTx`.